array-manipulation-1/src/tail.ts

```diff
@@ -0,0 +1,12 @@
+/* exported tail */
+function tail(array: unknown[]): unknown[] {
+  const tailArr = [];
+  for (let i = 0; i < array.length; i++) {
+    if (i === 0) {
+      continue;
```

Comment:
I get what you're thinking here of skipping 0, but you can just set `let i = 1`. There's not rule that we have to start at 0 and can start wherever we want, just like how you did it differently in `reverse.ts`